easy-rsa/Windows/revoke-full.bat

```diff
@@ -1,13 +1,13 @@
 @echo off
-cd %HOME%
+cd "%HOME%"
 rem revoke cert
-openssl ca -revoke %KEY_DIR%\%1.crt -config %KEY_CONFIG%
+openssl ca -revoke "%KEY_DIR%\%1.crt" -config "%KEY_CONFIG%"
 rem generate new crl
-openssl ca -gencrl -out %KEY_DIR%\crl.pem -config %KEY_CONFIG%
+openssl ca -gencrl -out "%KEY_DIR%\crl.pem" -config "%KEY_CONFIG%"
 rem test revocation
 rem first concatinate ca cert with newly generated crl
-copy %KEY_DIR%\ca.crt+%KEY_DIR%\crl.pem %KEY_DIR%\revoke_test_file.pem
+copy "%KEY_DIR%\ca.crt"+"%KEY_DIR%\crl.pem" "%KEY_DIR%\revoke_test_file.pem"
```

Comment:
According to [this page](http://www.easydos.com/copy.html) the "+" operator concatenates files together. Is this the correct way to quote this command?